syntax/parse.go

```diff
@@ -63,10 +63,9 @@ expr   -> C* amp="&"* @ C* arrow=(
         | C* "{:" C* embed=(grammar=@ ":" subgrammar=%%ast) ":}" C*
         | C* op="\\\\" @ C*
         | C* fn="\\" IDENT @ C*
-        | C* "//" pkg=( dot="."? ("/" name)+
-                      | "." std=IDENT?
-                      | http=/{https?://}? fqdn=name:"." ("/" path=name)*
-                      )
+		| C* "//" pkg=( "{" dot="."? STRING "}"
+				| std=IDENT?
+		)
```

Comment:
```suggestion
        | C* "//" pkg=( PKGPATH | std=IDENT? )
```
You'll need to update the logic to accommodate this.